### ISSUE TITLE
fix(testing): stabilize gradient control telemetry regression

### ISSUE THREAD

**Issue by ktinubu** [tinaudio/synth-setter#2839]
## What I saw

The focused finetune suite on current `main` intermittently fails despite `torch.manual_seed(1)`:

```text
tests/models/test_vst_flow_finetune_module.py::test_finetune_gradient_control_logs_positive_whole_batch_telemetry
assert metrics["train/control_cost"][0] > 0.0
E assert 0.0 > 0.0
```

## Expected

The regression fixture deterministically produces an active, nonzero gradient-control signal, or asserts a deterministic invariant that cannot collapse for a valid random draw.

## Reproduction

```bash
uv run pytest tests/models/test_vst_flow_finetune_module.py \
  -k gradient_control_logs_positive_whole_batch_telemetry -q
```

Observed while verifying the checkpoint-provenance split; no checkpoint-provenance code affects control generation.

Part of #2704

**Comment by ktinubu** [comment]
### [Claude-W7] Deterministic now — **15/15 green** — fixed by #2855, same repair as #2837

This and [#2837](https://github.com/tinaudio/synth-setter/issues/2837) are the same test collapsing on two different telemetry keys: `train/control_cost` here, `train/control_grad_norm` there. Both reported `assert 0.0 > 0.0`, which is what you get when the control signal is inactive for that draw rather than merely small.

At `b6d9cb08b6`, with the default random ordering and no seed pinning of my own:

```
tests/models/test_vst_flow_finetune_module.py -k gradient_control_logs_positive_whole_batch_telemetry
pass=15 fail=0
```

Fifteen runs rather than one, because this issue's claim is about *intermittency* — a single green run would not have contradicted it.

**And all three assertions survive**, so it is passing on the contract it was written for, not a relaxed one:

```python
assert metrics["train/control_signal_norm"][0] > 0.0
assert metrics["train/control_cost"][0] > 0.0        # the one reported here
assert metrics["train/control_grad_norm"][0] > 0.0   # the one reported in #2837
```

Fixed by **#2855** (`0f91938e5a`, 08-03, *pin control telemetry fixture time*), which `git log -L` over the test body identifies. That the repair was to pin the fixture's *time* fits both reports exactly: a degenerate `t` draw makes the control contribution vanish, zeroing cost and norm together, which is why one issue saw one key and one saw the other. Not a seeding problem, which is why `torch.manual_seed(1)` never helped.

Closing. Part of #2704.
